### PR TITLE
Add SAML Assertion Flow Grant blog

### PR DESCRIPTION
**Pull request by emanor-okta**
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [x] A title that's approved by Dev Advocacy
- [ ] A URL that's approved by Dev Advocacy
- [x] The content has been run through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2021-11-01-saml-assertion-flow-1.adoc [hunk 1]
```diff
@@ -0,0 +1,264 @@
+---
+layout: blog_post
+title: "OAuth SAML 2.0 Assertion Grant Type"
+author:
+by: advocate|contractor
```
Comment (havi) [inline _source/_posts/2021-11-01-saml-assertion-flow-1.adoc]:
```suggestion
by: internal-contributor
```
category for Okta staff writers not on dev-advocacy team

File: _source/_posts/2021-11-01-saml-assertion-flow-1.adoc [hunk 2]
```diff
@@ -0,0 +1,264 @@
+---
+layout: blog_post
+title: "OAuth SAML 2.0 Assertion Grant Type"
+author:
```
Comment (havi) [inline _source/_posts/2021-11-01-saml-assertion-flow-1.adoc]:
```suggestion
author: erik-manor
```

File: _source/_posts/2021-11-01-saml-assertion-flow-1.adoc [hunk 3]
```diff
@@ -0,0 +1,264 @@
+---
+layout: blog_post
+title: "OAuth SAML 2.0 Assertion Grant Type"
+author: erik-manor
```
Comment (aaronpk) [inline _source/_posts/2021-11-01-saml-assertion-flow-1.adoc:4]: Needs an entry for Erik in the [authors.yml](https://github.com/oktadev/okta-blog/blob/main/_source/_data/authors.yml) file

File: _source/_posts/2021-11-01-saml-assertion-flow-1.adoc [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+- ""
+image:
+type: awareness|conversion
+---
+
+:toc: macro
+:page-liquid:
+:experimental:
+
+Security Assertion Markup Language (SAML) is an open standard for exchanging authentication and authorization data between parties, in particular, between an identity provider and a service provider. The relationship between both providers sets up a trust relationship. The https://datatracker.ietf.org/doc/html/draft-ietf-oauth-saml2-bearer[SAML 2.0 Profile for OAuth 2.0 Client Authentication and Authorization Grants], allow an OAuth/OIDC client to use this existing trust. By providing a valid SAML Assertion to the authorization servers `/token` endpoint, the client can exchange an assertion for access/id tokens without requiring the client approval authorization step. This tutorial will demonstrate how to set up the grant type, then manually exchange a SAML Assertion for tokens. Afterward, an optional application can be configured to demonstrate the flow.
```
Comment (aaronpk) [inline _source/_posts/2021-11-01-saml-assertion-flow-1.adoc:21]: Can we avoid using the jargon terms "identity provider" and "service provider" in the opening sentence before we've had a chance to define them?

We also need a definition of "SAML Assertion" before assuming the reader knows what it is or how they would get one.

File: _source/_posts/2021-11-01-saml-assertion-flow-1.adoc [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+== Reference Okta Org
+A preconfigured Okta Org exists so settings can be verified.   
+To view the configurations, login to https://dev-96797994-admin.okta.com[https://dev-96797994-admin.okta.com] +
+Credentials +
+Username: `read.only` +
+Password: `Th1sPassword` +
+
+== Configure the SAML Service Provider Application
+The same Okta Org will both generate and consume a SAML Assertion. So the OIDC application will consume a SAML Assertion from the same Org it is defined in. To accomplish this a SAML Service Provider application using `\https://httpbin.org/post` will be created. +
```
Comment (aaronpk) [inline _source/_posts/2021-11-01-saml-assertion-flow-1.adoc:38]: Add a note that this is not a normal workflow, just for this demonstration.

File: _source/_posts/2021-11-01-saml-assertion-flow-1.adoc [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+* https://golang.org/dl/[Golang] (1.16+) **if running the sample application**.
+
+== Reference Okta Org
+A preconfigured Okta Org exists so settings can be verified.   
+To view the configurations, login to https://dev-96797994-admin.okta.com[https://dev-96797994-admin.okta.com] +
+Credentials +
+Username: `read.only` +
+Password: `Th1sPassword` +
+
+== Configure the SAML Service Provider Application
```
Comment (aaronpk) [inline _source/_posts/2021-11-01-saml-assertion-flow-1.adoc:37]: We need some sort of definition of the roles involved before diving in. For example what is a "Service Provider"?

**Review by aaronpk — CHANGES_REQUESTED** [review]

**Comment by mraible** [comment]
@emanor-okta Is this a post you'd still like to publish?

**Comment by edunham** [comment]
Checked with emanor via Slack and the desired disposition of this PR is closing it out.

**Pull request closed without merging**